api/v1alpha1/clusterextension_types.go

```diff
@@ -83,6 +83,11 @@ type ClusterExtensionSpec struct {
 	//+kubebuilder:Optional
 	// Preflight defines the configuration of preflight checks.
 	Preflight *PreflightConfig `json:"preflight,omitempty"`
+
+	//+kubebuilder:validation:MaxLength:=48
+	//+kubebuilder:validation:Pattern:=^[a-z0-9]+(-[a-z0-9]+)*$
+	// ServiceAccount is used to install and manage content
+	ServiceAccount string `json:"serviceAccount"`
```

Comment:
I followed up to figure out what a good convention should be for a reference to a ServiceAccount.

I propose we follow the OpenShift convention: https://github.com/openshift/enhancements/blob/master/dev-guide/api-conventions.md#use-specific-types-for-object-references-and-omit-ref-suffix, which would mean introducing a new `ServiceAccountReference` type that includes a `Name string` field.